servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PublishAndSubscribeOnCompletables.java

```diff
@@ -110,8 +107,11 @@ void handleSubscribe(final Subscriber subscriber,
                 Subscriber wrapped = contextProvider.wrapCancellable(subscriber, contextMap);
```

Comment:
(can't comment on `AbstractAsynchronousPublisherOperator`). I just noticed that `AbstractAsynchronousPublisherOperator` allows you to override `handleSubscribe` however the overriding class to un-do the AsyncContext wrapping. We should make `AbstractAsynchronousPublisherOperator#handleSubscribe` final (just like `AbstractSynchronousPublisherOperator`) to avoid this confusion.